python/lsst/ts/mtmount/mtmount_csc.py

```diff
@@ -938,9 +942,9 @@ async def retry_command(
         ----------
         command : `commands.Command`
             Class of command to send
-        max_tries : `int`
+        max_tries : `int`, optional
             Maximum number of times to issue the command.
-        timeout : `float`
+        timeout : `float`, optional
             Timeout (sec) for the command (applied to each retry).
             Keep it short to avoid blocking the command lock for a long time.
```

Comment:
Maybe put the default value in the doc string for the optional argument?

Reply:
I try to avoid that unless it's really important, because It shows up in the built docs without that, and I prefer not to duplicate information.